README.md

```diff
@@ -312,92 +312,94 @@ For more on Grist Labs' history and principles, see our [About Us](https://www.g
+| GRIST_UI_FEATURES                               | comma-separated list of UI features to enable. Allowed names of parts: `helpCenter`, `billing`, `templates`, `createSite`, `multiSite`, `multiAccounts`, `sendToDrive`, `tutorials`, `supportGrist`, `themes`. If a part also exists in GRIST_HIDE_UI_ELEMENTS, it won't be enabled.                                                                                                                                                                       |
+| GRIST_UNTRUSTED_PORT                            | if set, plugins will be served from the given port. This is an alternative to setting APP_UNTRUSTED_URL.                                                                                                                                                                                                                                                                                                                                                   |
+| GRIST_WIDGET_LIST_URL                           | a url pointing to a widget manifest, by default https://github.com/gristlabs/grist-widget/releases/download/latest/manifest.json is used                                                                                                                                                                                                                                                                                                                   |
+| GRIST_LOG_HTTP                                  | When set to `true`, log HTTP requests and responses information. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                                      |
+| GRIST_LOG_HTTP_BODY                             | When this variable and `GRIST_LOG_HTTP` are set to `true` , log the body along with the HTTP requests. :warning: Be aware it may leak confidential information in the logs.:warning: Defaults to `false`.                                                                                                                                                                                                                                                  |
+| GRIST_LOG_AS_JSON                               | When this variable is set to `true` or a truthy value, log in JSON.                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
(I think this section of the README got worse when whitespace was added :(. It isn't great but at least it used to be trivial to edit and review. Still not hard, just annoying, and tooling dependent.  I wouldn't mind if someone got rid of all the padding again)

Could you say "output log lines in JSON as opposed to a plain text format" just to be 100% clear on what changes?

Reply:
Done and done!